st2api/st2api/controllers/v1/webhooks.py

```diff
@@ -51,11 +51,12 @@ def __init__(self, *args, **kwargs):
         self._trigger_types = WEBHOOK_TRIGGER_TYPES.keys()
 
         self._trigger_dispatcher = TriggerDispatcher(LOG)
+        queue_suffix = self.__class__.__name__ + ':' + str(uuid.uuid4())
```

Comment:
Another minor refactoring thing - we could potentially refactor that function and add another argument `append_random_id` or similar and leverage that.